app/views/early_career_payments/claims/doctoral_loan.html.erb

```diff
@@ -0,0 +1,48 @@
+<% content_for(:page_title, page_title(t("early_career_payments.questions.postgraduate_doctoral_loan"), policy: current_policy_routing_name, show_error: current_claim.errors.any?)) %>
+<% path_for_form = current_claim.persisted? ? claim_path(current_policy_routing_name) : claims_path(current_policy_routing_name) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: current_claim, errored_field_id_overrides: { "eligibility.postgraduate_doctoral_loan": "claim_eligibility_attributes_postgraduate_doctoral_loan_true" }) if current_claim.errors.any? %>
+    <%= form_for current_claim, url: path_for_form do |form| %>
+      <%= form_group_tag current_claim do %>
+        <%= form.fields_for :eligibility, include_id: false do |fields| %>
+
+          <%= fields.hidden_field :postgraduate_doctoral_loan %>
+
+          <fieldset class="govuk-fieldset">
+
+            <span class="govuk-caption-xl">About your student loans</span>
```

Comment:
@martikat Just looking at the layouts in the prototype. I think this needs to be outside of the <fieldset> tag? Can you double check please?